default/python/universe_generation/galaxy.py

```diff
@@ -45,32 +307,10 @@ def calc_universe_width(shape, size):
     return width
 
 
-def get_systems_within_jumps(origin_system, jumps):
-    """
-    Returns all systems within jumps jumps of system origin_system (including origin_system).
-    If jumps is 0, return list that only contains system origin_system.
-    If jumps is negative, return empty list.
-    """
-    if jumps < 0:
-        return []
-    matching_systems = [origin_system]
-    next_origin_systems = [origin_system]
-    while jumps > 0:
-        origin_systems = list(next_origin_systems)
-        next_origin_systems = []
-        for system in origin_systems:
-            neighbor_systems = [s for s in fo.sys_get_starlanes(system) if s not in matching_systems]
-            next_origin_systems.extend(neighbor_systems)
-            matching_systems.extend(neighbor_systems)
-        jumps -= 1
-    return matching_systems
```

Comment:
Although it's only used be the `empire.py` module, this function has been deliberatly placed here in the `galaxy.py` module. After all, this module is where all the stuff that deals with the galaxy map belongs, and this function is provided not only for what it's used in `empire.py`, but potential future uses in other modules as well (e.g. users who want to customize/tweak the universe generation scripts might need it for whatever things the want to accomplish).

So IMO it should stay here.